diagnostics/forcing_feedback/forcing_feedback_plot.py

```diff
@@ -0,0 +1,522 @@
+# This file is part of the forcing_feedback module of the MDTF code package (see mdtf/MDTF_v2.0/LICENSE.txt)
+
+# ======================================================================
+# forcingfluxanom_plot_final.py
+#
+#  Called by forcingfluxanom_xr_final.py
+#   Reads in observations and temporary model results of individual 2D feedbacks and IRF, produces maps of each and
+#   a bar graph summarizing global-mean values
+#
+#  Forcing Feedback Diagnositic Package
+#
+#  This file is part of the Forcing and Feedback Diagnostic Package
+#    and the MDTF code package. See LICENSE.txt for the license. 
+#
+#
+
+import os
+import sys
+import numpy as np
+import numpy.ma as ma
+import xarray as xr
+import cartopy as car
+import matplotlib as mpl
+mpl.use('Agg')
+import matplotlib.pyplot as plt
+from matplotlib import colors as c
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import matplotlib.ticker as mticker
+import cartopy.crs as ccrs
+
+from cartopy.mpl.gridliner import LONGITUDE_FORMATTER, LATITUDE_FORMATTER
+#from mpl_toolkits.basemap import Basemap, shiftgrid
+
+def globemean_2D(var,w):
+    var_mask = ma.masked_array(var,~np.isfinite(var))
+    var_mean = np.squeeze(np.average(np.nanmean(var_mask,axis=1),weights=w))
+
+    return var_mean
+
+# Read in observational data
+nc_obs = xr.open_dataset(os.environ["OBS_DATA"]+"/forcing_feedback_obs.nc")
+lat_obs = nc_obs.lat.values
+lon_obs = nc_obs.lon.values
+llons_obs, llats_obs = np.meshgrid(lon_obs,lat_obs)
+
+#Read in model results
+
+nc_pl = xr.open_dataset(os.environ["WK_DIR"]+"/model/fluxanom2D_Planck.nc")
+nc_lr = xr.open_dataset(os.environ["WK_DIR"]+"/model/fluxanom2D_LapseRate.nc")
+nc_lw_q = xr.open_dataset(os.environ["WK_DIR"]+"/model/fluxanom2D_LW_WaterVapor.nc")
+nc_sw_q = xr.open_dataset(os.environ["WK_DIR"]+"/model/fluxanom2D_SW_WaterVapor.nc")
+nc_alb = xr.open_dataset(os.environ["WK_DIR"]+"/model/fluxanom2D_SfcAlbedo.nc")
+nc_lw_c = xr.open_dataset(os.environ["WK_DIR"]+"/model/fluxanom2D_LW_Cloud.nc")
+nc_sw_c = xr.open_dataset(os.environ["WK_DIR"]+"/model/fluxanom2D_SW_Cloud.nc")
+nc_lw_irf = 12*xr.open_dataset(os.environ["WK_DIR"]+"/model/fluxanom2D_LW_IRF.nc")
```

Comment:
Haven't tested this, but assuming it works as intended (i.e., multiplying an xarray DataSet by a scalar will result in only the dependent variable arrays being scaled, with coordinates unchanged.) It may be more straightforward to just generate the data in fluxanom2D_LW_IRF.nc as annual, rather than multiplying it here.